src/java.management/share/classes/com/sun/jmx/remote/security/MBeanServerFileAccessController.java

```diff
@@ -301,13 +300,11 @@ private static Properties propertiesFromFile(String fname)
     }
 
     private synchronized void checkAccess(AccessType requiredAccess, String arg) {
-        @SuppressWarnings("removal")
-        final AccessControlContext acc = AccessController.getContext();
         @SuppressWarnings("removal")
         final Subject s =
             AccessController.doPrivileged(new PrivilegedAction<>() {
                     public Subject run() {
-                        return Subject.getSubject(acc);
+                        return Subject.current();
```

Comment:
Is the `doPrivileged` still needed here? Is there a chance that Subject.current() will throw a SecurityException, or return a different result if a security manager is present?